third_party/pyth/price-service/src/listen.ts

```diff
@@ -194,15 +211,21 @@ export class Listener implements PriceStore {
       ) {
         const priceFeed = priceAttestationToPriceFeed(priceAttestation);
         const priceInfo = {
-          seqNum: parsedVAA.sequence,
-          vaaBytes,
+          seqNum: parsedVaa.sequence,
+          vaa,
           attestationTime: priceAttestation.attestationTime,
           priceFeed,
-          emitterChainId: parsedVAA.emitter_chain,
+          emitterChainId: parsedVaa.emitter_chain,
           priceServiceReceiveTime: Math.floor(new Date().getTime() / 1000),
         };
         this.priceFeedVaaMap.set(key, priceInfo);
 
+        if (lastAttestationTime !== undefined) {
+          this.promClient?.addPriceUpdatesGap(
+            priceAttestation.attestationTime - lastAttestationTime
+          );
+        }
```

Comment:
should we also have a similar metric that tracks the gap between the last price update times as well? That would be more of an end-to-end performance metric.

Reply:
Also let's name this histogram something like "attestationGap" to distinguish between the two cases

Reply:
Good idea 👍